src/Capgemini.PowerApps.PackageDeployerTemplate/Adapters/TraceLoggerAdapter.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Xrm.Tooling.PackageDeployment.CrmPackageExtentionBase;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Capgemini.PowerApps.PackageDeployerTemplate.Adapters
+{
+    internal class TraceLoggerAdapter : ILogger
+    {
+        private readonly TraceLogger traceLogger;
+
+        private readonly Dictionary<LogLevel, TraceEventType> logLevelMap = new Dictionary<LogLevel, TraceEventType>
```

Comment:
Really minor but I think this ought to be `static`

Reply:
I assume you meant only `logLevelMap`?

Reply:
Yep 🙂